dashboard/db/migrate/20170822201210_backfill_authorized_teacher_user_permissions.rb

```diff
@@ -0,0 +1,17 @@
+class BackfillAuthorizedTeacherUserPermissions < ActiveRecord::Migration[5.0]
+  def up
+    Plc::UserCourseEnrollment.find_each do |enrollment|
+      UserPermission.create(permission: UserPermission::AUTHORIZED_TEACHER, user_id: enrollment.user_id)
+    end
+
+    Cohort.find_each do |cohort|
+      cohort.teachers.each do |teacher|
+        UserPermission.create(permission: UserPermission::AUTHORIZED_TEACHER, user_id: teacher.id)
+      end
+    end
+  end
+
+  def down
+    UserPermission.where(permission: UserPermission::AUTHORIZED_TEACHER).destroy_all
```

Comment:
Given that we may have created some `AUTHORIZED_TEACHER` permissions manually between the `up` and the `down`, does it make sense to destroy them all? My expectation would have been to use the default (empty) `down` method.